libs/external-data/src/lib/airtable/index.ts

```diff
@@ -357,6 +358,36 @@ export class AirtableClient {
       })) as PageData[];
   }
 
+  async getAnnotations(lastUpdatedDate?: DateType): Promise<AnnotationsData[]> {
+    const params = lastUpdatedDate
+      ? {
+          filterByFormula: createLastUpdatedFilterFormula(lastUpdatedDate),
+        }
+      : {};
+    const query = this.createQuery(bases.ANNOTATIONS, 'Events', params);
+
+    return (await this.selectAll(query))
+      .filter(({ fields }) => Object.values(fields).some((value) => value))
+      .map(({ id, fields }) => ({
+        airtable_id: id,
+        title: squishTrim(fields['Title']),
+        title_fr: squishTrim(fields['Title FR']),
+        event_type: squishTrim(fields['Event Type']),
+        description: squishTrim(fields['Description']),
+        description_fr: squishTrim(fields['Description FR']),
+        event_date: dayjs.utc(fields['Event Date']).toDate(),
+        data_affected: fields['Data affected']?.map(squishTrim),
+        tasks_affected: fields['Tasks affected']?.map(squishTrim),
+        audience: fields['Audience']?.map(squishTrim),
+        entered_by: squishTrim(fields['Entered by']),
+        date_entered: dayjs.utc(fields['Date entered']).toDate(),
```

Comment:
```suggestion
        date_entered: fields['Date entered'] && dayjs.utc(fields['Date entered']).toDate(),
```
Because this field will become optional. If you're not familiar with the pattern, it's more or less a type of null-check.
If the first value is falsy -> it returns the first value
If the first value is truthy -> it returns the second value

In other words, if the value is missing, it'll be undefined, and if it's not missing we'll move on and convert it to a date.

Otherwise we'll be passing `undefined` to the dayjs constructor and end up with a date of January 1st 1970 instead of an empty value.